packages/aggregator/src/aggregationTypes.js

```diff
@@ -4,28 +4,29 @@
  */
 
 export const AGGREGATION_TYPES = {
-  RAW: 'RAW',
-  MOST_RECENT: 'MOST_RECENT',
-  MOST_RECENT_PER_ORG_GROUP: 'MOST_RECENT_PER_ORG_GROUP',
-  SUM: 'SUM',
-  SUM_UNTIL_CURRENT_DAY: 'SUM_UNTIL_CURRENT_DAY',
-  SUM_MOST_RECENT_PER_FACILITY: 'SUM_MOST_RECENT_PER_FACILITY',
-  FINAL_EACH_DAY: 'FINAL_EACH_DAY',
+  COUNT_PER_ORG_GROUP: 'COUNT_PER_ORG_GROUP',
```

Comment:
Those were sorted alphabetically here + the new type was added (see commit history)